tests/emukit/core/test_parameter_space.py

```diff
@@ -38,6 +38,17 @@ def test_check_in_domain(space_2d, space_3d_mixed):
     assert np.array_equal(in_domain, np.array([False, True]))
 
 
+def test_check_in_domain_with_bandit_parameter():
```

Comment:
Can you also add an isolated unit test, for a domain with bandit-only parameters?

Reply:
Sorry for the delay!